.github/workflows/sync-kubernetes-configuration-crds.yaml

```diff
@@ -35,10 +35,17 @@ jobs:
           /tmp/kustomize build /tmp/kubernetes-configuration > charts/gateway-operator/charts/kubernetes-configuration-crds/crds/kubernetes-configuration-crds.yaml
 
       - name: Create Pull Request
-        uses: peter-evans/create-pull-request@67ccf781d68cd99b580ae25a5c18a1cc84ffff1f
+        uses: peter-evans/create-pull-request@dd2324fc52d5d43c699a5636bcf19fceaa70c284
         with:
-          title: 'chore: update kubernetes-configuration CRDs to ${{ github.event.inputs.version }}'
+          token: ${{ secrets.K8S_TEAM_BOT_GH_PAT }}
+          title: Sync kubernetes-configuration CRDs to ${{ github.event.inputs.version }}
           commit-message: 'chore: update kubernetes-configuration CRDs to ${{ github.event.inputs.version }}'
-          branch: chore/update-kubernetes-configuration-crds
+          committer: Kong's Team k8s bot <team-k8s+github-bot@konghq.com>
+          author: Kong's Team k8s bot <team-k8s+github-bot@konghq.com>
+          signoff: true
+          base: main
```

Comment:
This should be OK for now but in case we ever want to introduce release branches, this will have to be updated.

I think we could use `github.ref`, defaulting to `main` if it's unavailable, WDYT?

Reply:
SGTM. Addressed.